pkg/apis/serving/v1beta1/configmap.go

```diff
@@ -31,11 +31,12 @@ import (
 
 // ConfigMap Keys
 const (
-	ExplainerConfigKeyName = "explainers"
-	IngressConfigKeyName   = "ingress"
-	DeployConfigName       = "deploy"
-	LocalModelConfigName   = "localModel"
-	SecurityConfigName     = "security"
+	InferenceServiceConfigKeyName = "inferenceService"
```

Comment:
I'm seeing here `S` in uppercase, but the manifest example in `inferenceservice.yaml` is all lowercase. Should the case match?

Reply:
Nice catch, the test didn't fail because I used the value in the camel case as defined here.
updated the example in the yaml.